From c9abbec48a2c1db5a88a9fb60fc350514af3dcf2 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Fri, 20 Jul 2018 12:22:50 +0200
Subject: [PATCH] build: Fix build with newer meson

The gtk-doc meson helper regressed in that it didn't expect the
libfprint.types file to not exist, as earlier versions did. Remove
the --rebuild-types call so that it is not generated, and meson carries
on compiling the rest of the library.

See https://gitlab.gnome.org/GNOME/gtk-doc/issues/48
See https://github.com/mesonbuild/meson/issues/3892
See https://bugzilla.redhat.com/show_bug.cgi?id=1604585
---
 doc/meson.build | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/doc/meson.build b/doc/meson.build
index 48358c8..c2c1b77 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -54,7 +54,6 @@ gnome.gtkdoc('libfprint',
              content_files: content_files,
              expand_content_files: expand_content_files,
              scan_args: [
-               '--rebuild-types',
                '--ignore-decorators=API_EXPORTED',
                '--ignore-headers=' + ' '.join(private_headers),
              ],
-- 
2.17.1

